src/pages/documentation/adobe-target/event-reference.md

```diff
@@ -0,0 +1,200 @@
+
+The Target extension will read the following keys from the configuration event:
+
+| **Key** | **Value Type** | **Required** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `target.clientCode` | String | Yes | The Target Client Code that is obtained during provisioning with the Target Adobe solution. |
+| `target.environmentId` | Integer | No | The Target Environment ID. |
+| `target.timeout` | Integer | No | This value is used for Target request connection timeouts (in seconds). If the value is not set, a default value of 5 seconds is used. |
+| `global.privacy` | String | No | This contains the mobile privacy status settings. |
+| `target.previewEnabled` | Boolean | No | Boolean value to determine whether Target preview should be enabled. If not set, then the preview is enabled by default. |
+
+## Events dispatched
+
+The following events are dispatched by the Target extension:
+
+### Target Content response
+
+This event is created when the Target server response is received, and the prefetch or load request operation is completed. Network responses are generated from the Target server for prefetch or load request calls.
+
+This event is dispatched in the following scenarios:
+
+* In response to an API prefetch content request. This event contains a `prefetchresult` key of type _boolean_, which signals the success or failure of the prefetch request.
+* In response to an API batch load requests, where one event is dispatched per request.
+
+  This event contains a `content` key of type _string_ that contains mbox content.
+
+#### Data payload definition
+
+Here are the key-value pairs in this event:
+
+| **Key** | **Value Type** | **Required** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `content` | String | No | Response content is returned by the Target server for a load request with valid locations. |
+| `prefetchresult` | Boolean | No | Prefetch status returned after a Target prefetch request. |
+| `prefetcherror` | String | No | Prefetch response error if any, otherwise null |
+
+### Target Identity response
+
+This event delivers the Target identities to the following:
+
+* The original requester.
+* To any extension that might be listening for updates.
+
+This event is generated by the Target extension as a response to the Target Identity Request event.
+
+#### Data payload definition
+
+Here are the key-value pairs in this event:
+
+| **Key** | **Value Type** | **Required** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `tntid` | String | No | The Target Identifier retrieved from Target server. |
+| `thirdpartyid` | String | No | Current Third Party ID that is set from the API. |
+
+### Target Preview response
+
+This event is generated when Target Preview is started or stopped by the marketer. It can be used to trigger any custom workflows or API calls based on the preview flag by the following:
```

Comment:
These first sentence sounds a bit weird when I read it... maybe it would sound better as "This event is generated when the marketer enters or exits Target preview mode." What do you think?

Reply:
Definitely reads a lot better. I'll update it to that instead.